scripts/_helpers.py

```diff
@@ -22,6 +23,40 @@
 
 REGION_COLS = ["geometry", "name", "x", "y", "country"]
 
+logger = logging.getLogger(__name__)
+
+
+def handle_exception(exc_type, exc_value, exc_traceback):
+    tb = exc_traceback
+    while tb.tb_next:
+        tb = tb.tb_next
+    flname = tb.tb_frame.f_globals.get("__file__")
+    funcname = tb.tb_frame.f_code.co_name
+
+    if issubclass(exc_type, KeyboardInterrupt):
+        logger.error(
+            "Manual interruption %r, function %r: %s",
+            flname,
+            funcname,
+            exc_value,
+        )
+    else:
+        logger.error(
+            "An error happened in module %r, function %r: %s",
+            flname,
+            funcname,
+            exc_value,
+            exc_info=(exc_type, exc_value, exc_traceback),
+        )
+
+
+def create_logger(level=logging.INFO):
+    # logger = logging.getLogger('__main__.' + __name__)
+    logger = logging.getLogger("__main__")
```

Comment:
I think we are quite close :D

I'd add a first mandatory input to this function like in the following.
Then, it can be used flexibly to create arbitrary loggers. This may solve the issue you mentioned in build shapes.
Could you test if it works?

```suggestion
def create_logger(logger_name, level=logging.INFO):
    logger = logging.getLogger(logger_name)
```

Finally, docstrings may be recommendede to add, even simple ones with no ful description of inputs.
I'd say that the create_logger shall have a proper docstring, while handle_exception can have a simpler one

Reply:
> I think we are quite close :D
> 
> I'd add a first mandatory input to this function like in the following. Then, it can be used flexibly to create arbitrary loggers. This may solve the issue you mentioned in build shapes. Could you test if it works?
> 
> Finally, docstrings may be recommendede to add, even simple ones with no ful description of inputs. I'd say that the create_logger shall have a proper docstring, while handle_exception can have a simpler one

It works!!!
Thank you so much :D
Adding docstrings :)